content/product/iac.md

```diff
@@ -0,0 +1,241 @@
+            - title: __main__.py
+              language: python
+              code: |
+                import pulumi
+                import pulumi_eks as eks
+
+                # Create an EKS cluster with the default configuration.
+                cluster = eks.Cluster("eks-cluster")
+
+                # Export the cluster's kubeconfig.
+                pulumi.export("kubeconfig", cluster.kubeconfig)
+            - title: main.go
+              language: go
+              code: |
+                    package main
+
+                    import (
+                      "github.com/pulumi/pulumi-eks/sdk/go/eks"
+                      "github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+                    )
+
+                    func main() {
+                      pulumi.Run(func(ctx *pulumi.Context) error {
+                        // Create an EKS cluster with default settings.
+                        cluster, err := eks.NewCluster(ctx, "eks-cluster", nil)
+                        if err != nil {
+                          return err
+                        }
+
+                        // Export the cluster's kubeconfig.
+                        ctx.Export("kubeconfig", cluster.Kubeconfig)
+                        return nil
+                      })
+                    }
+            - title: MyStack.cs
+              language: csharp
+              code: |
+                using System.Collections.Generic;
+                using Pulumi;
+                using Pulumi.Eks;
+
+                await Deployment.RunAsync(() =>
+                {
+                  // Create an EKS cluster with default settings.
+                  var cluster = new Cluster("eks-cluster");
+
+                  // Export the cluster's kubeconfig.
+                  return new Dictionary<string, object?>
+                  {
+                    ["kubeconfig"] = cluster.Kubeconfig
+                  };
+                });
+            - title: Main.Java
+              language: java
+              code: |
+                import com.pulumi.Context;
+                import com.pulumi.Pulumi;
+                import com.pulumi.eks.Cluster;
+
+                public class App {
+                    public static void main(String[] args) {
+                        Pulumi.run(App::stack);
+                    }
+
+                    private static void stack(Context ctx) {
+                    final var cluster = new Cluster("eks-cluster");
+                    ctx.export("kubeconfig", cluster.kubeconfig());
+                  }
+                }
+            - title: Pulumi.yaml
+              language: yaml
+              code: |
+                resources:
+                  eks-cluster:
+                    type: eks:Cluster
+                outputs:
+                  kubeconfig: ${cluster.kubeconfig}
+          button:
+            text: "Learn more about Pulumi Packages"
+            link: "/product/packages/"
```

Comment:
oh wow this is an orphaned page. 

should we be putting them to this page instead? https://www.pulumi.com/docs/using-pulumi/pulumi-packages/

Reply:
This shouldn't be an orphaned page. https://www.pulumi.com/product/packages/ exits and is being linked to from the Pulumi Overview page rn.